src/schema/v2/artist/index.ts

```diff
@@ -316,9 +316,11 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
             sizes,
             sort: options.sort,
             state: options.state,
-            sale_start_year: options.saleStartYear,
-            sale_end_year: options.saleEndYear,
             allow_unspecified_sale_dates: options.allowUnspecifiedSaleDates,
+            ...(options.saleStartYear && {
```

Comment:
Obviously, if we notice a repeating pattern, we could create a method for omitting null values.